### DISCUSSION TITLE
[Enhancement] Widgets / LiveTiles

### DISCUSSION THREAD

**Discussion by mhrastegari** [dotnet/maui#156] [General]
## Summary
so iOS 14 announced and has the liveTiles now(i know 😂) 
and i think it's time and would be really nice to support them.

## Platforms
UWP: LiveTiles
iOS: Home Screen Widgets
macOS: Notification Center Widgets
Android: Widgets

**Comment by newcc845** [comment]
Hope this widget features will available in MAUI soon.

**Reply by manishbisen** [reply]
Yes,  but when, nobody knows.
